Lombiq.OrchardCoreApiClient/ApiClient.cs

```diff
@@ -43,24 +47,18 @@ public ApiClient(ApiClientSettings apiClientSettings)
 
         try
         {
-            (await OrchardCoreApi.
-                CreateAsync(createApiViewModel).
-                ConfigureAwait(false)).
-                ResponseMessage.EnsureSuccessStatusCode();
```

Comment:
How does Refit provides this behavior?

Reply:
Refit automatically throws an `ApiException` for `Task<T>` returning methods if the response is not successful. ([see docs](https://github.com/reactiveui/refit#when-returning-taskt)) Actually if the method returns `Task<ApiResponse<T>>` then the same exception is captured, so that's just extra busywork. By the way [RestEase does the same as well](https://github.com/canton7/RestEase#response-status-codes) so this was never necessary.

By the way both Refit's `ApiResponse<T>` and RestEase's `Response<T>` are `IDisposable` so it's good to avoid them unless they are really necessary. The original code didn't dispose the results here so that was a memory leak.